src/handlers/list.ts

```diff
@@ -1,17 +1,20 @@
 import { readFile } from "node:fs/promises";
 import { Task } from "../models/Task";
 
+// it handles list commands
```

Comment:
Great idea of documenting what the namespaces do! But I wonder if this message is enough for new developers... Maybe we could rewrite it to make it clearer?

Reply:
That's a very good point! I will!